Dockerfile

```
@@ -0,0 +1,8 @@
+FROM node:12
+
+WORKDIR /usr/src/app
+COPY package*.json ./
+
+RUN npm install
+
```

```suggestion

EXPOSE 3000

```
You can expose the app port for greater ease of use.